### ISSUE TITLE
Text unreadable on HiDPI screens

### ISSUE THREAD

**Issue by Lucretia** [brummer10/guitarix#111]
The labels in the tree roots, i.e. "Tone control" are tiny and need increasing.

**Comment by brummer10** [comment]
Could you post a screenshot please. 
Scaling on HiDPI is normally done by GTK.

**Comment by Lucretia** [comment]
![Screenshot_20220526_135301](https://user-images.githubusercontent.com/570928/170491860-7ed260a5-1433-4a38-beb3-f8d1608b0355.png)

**Comment by brummer10** [comment]
Strange. I've no idea how that comes. When I tried here with GDK_SCALE the label in the tool-palette scale the same as all others.

**Comment by szszoke** [comment]
Could this be something caused by running the application in KDE?

**Comment by Lucretia** [comment]
> Could this be something caused by running the application in KDE?

No, KDE has some sort of Gtk compat built in to allow Gtk apps to work. This is most likely a 4K thing as most UI issues seems to be 4K issues.

**Comment by forrestkuen** [comment]
have you found a solution for the problem with the tiny letters?
